tests/e2e/test_as_tenant.yml

```diff
@@ -118,6 +120,8 @@
     #
     # * boot_disk_copy: true
     #
+    # * data_copy: true
+    #
```

Comment:
This part looks good but we should also add a 3rd workload migration that would set "data_copy: false"